webgrid/__init__.py

```diff
@@ -326,8 +337,25 @@ def render_html(self, record, hah):
         data = self.extract_and_format_data(record)
         if not data:
             return data
+        # if we have an arrow date, allow html_format to use that functionality
+        if arrow and isinstance(data, arrow.Arrow):
+            if data.strftime(self.html_format) == self.html_format:
+                return data.format(self.html_format)
         return data.strftime(self.html_format)
 
+    def render_xls(self, record):
+        data = self.extract_and_format_data(record)
+        if not data:
+            return data
+        # if we have an arrow date, pull the underlying datetime, else the renderer won't know
+        #   how to handle it
+        if arrow and isinstance(data, arrow.Arrow):
+            data = data.datetime
+        # xlwt has no idea what to do with zone information
+        if isinstance(data, dt.datetime) and data.tzinfo is not None:
+            data = data.replace(tzinfo=None)
```

Comment:
Does this just strip the TZ info or does it convert it to UTC? It would be best if we always gave Excel UTC.